index.html

```diff
@@ -158,17 +160,31 @@
     d.isPr ? inlineApp.data.allPrs += 1 : inlineApp.data.allIssues += 1 ;
     if (d.assignees.length == 0) {
       inlineApp.data.issues[inlineApp.NOT_ASSIGNED].push(d)
+      inlineApp.data.storypoint[inlineApp.NOT_ASSIGNED] += inlineApp.retrieveStoryPoint(d);
     } else {
       d.isPr ? inlineApp.data.assignedPrs += 1 : inlineApp.data.assignedIssues += 1 ;
       for (var i = 0; i < d.assignees.length; i++) {
         var assignee = d.assignees[i].login
         if (!inlineApp.data.issues[assignee]) {
           inlineApp.data.issues[assignee] = [];
+          inlineApp.data.storypoint[assignee] = 0;
         }
         inlineApp.data.issues[assignee].push(d);
+        inlineApp.data.storypoint[assignee] += inlineApp.retrieveStoryPoint(d);
       }
     }
   },
+  retrieveStoryPoint: function(issue) {
+    var point = 0;
+    // treat as storypoint if label's name is only digits.
+    for (var j = 0; j < issue.labels.length; j++) {
+      label = issue.labels[j];
+      if (label.name.match(/^[0-9]+$/)) {
+         point += +label.name;
+      }
```

Comment:
"1/2" should be also treated as story point. 
How about adding `else if (label.name === '1/2') { point += 0.5; }` ?